.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+# Notes:
+#  - All software installed below is pinned so they don't get upgraded and break w/o us knowing why.
+#  - If you bump one version up to latest, please bump them all so we stay current!
+
+name: Build medic-couch2pg and run unit test cases
+
+on: [push, pull_request]
+
+jobs:
+  build-and-test:
+    name: Build medic-couch2pg
+    runs-on: ubuntu-18.04
+
+    steps:
+    - name: Checkout branch
+      uses: actions/checkout@v2
+
+    - name: Use nodeJS 10.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+
+    - name: Set environment variables
+      run: |
+        echo "COUCH2PG_SLEEP_MINS=120" >> $GITHUB_ENV
+        echo "COUCH2PG_DOC_LIMIT=1000" >> $GITHUB_ENV
+        echo "COUCH2PG_RETRY_COUNT=5" >> $GITHUB_ENV
```

Comment:
Are these env vars needed in multiple steps? If not, please move them to the step that needs them with the more "normal" `env` syntax. If they are needed in multiple, the duplication is undesirable and I was surprised to see that this is indeed the recommended approach, in which case this is fine as-is.

Reply:
Fixed! I found a nicer way of setting these inside the step, instead of global